onboarding/web-infra.md

```diff
@@ -0,0 +1,23 @@
+# Web Infrastructure Onboarding
+
+This document is an outline of the tasks necessary to onboard a new member of the Web Infrastructure team.
+
+Please note these tasks should only be performed after the person's nomination has passed.
+
+## Tasks
+
+Tasks prefixed by `Member:` are for the new member to complete.
+Tasks prefixed by `Web Admin:` are for a member of `@nodejs/web-admins` to complete.
+
+- [ ] Web Admin: Create an issue in [nodejs/web-team](https://github.com/nodejs/web-team) specifically for onboarding the member and include this list in it.
```

Comment:
Honestly think this might be better as an issue template and this doc just links to it, wdyt?

Reply:
I think the Member can do this in their "Add xyz to the list of MEMBERS" PR, and the team will check off each task before merging?